app/src/main/java/org/mozilla/rocket/content/ChromeViewModel.kt

```diff
@@ -0,0 +1,78 @@
+package org.mozilla.rocket.content
+
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.ViewModel
+import org.mozilla.focus.utils.Settings
+import org.mozilla.rocket.download.SingleLiveEvent
+
+class ChromeViewModel(settings: Settings) : ViewModel() {
+    val isNightMode = MutableLiveData<Boolean>()
+    val tabCount = MutableLiveData<Pair<Int, Boolean>>()
```

Comment:
You can either name the live data to something like tabCountAnimated, or simply use some data class other than Pair. Just to make sure the receiver knows what this boolean stands for.

Reply:
created the data class `TabCountChangedEvent`